server/helpers/userServer.js

```diff
@@ -1,14 +1,17 @@
 /* eslint-disable */

```

Comment:
@jo-rdan  do not disable any Eslint flag. the purpose of using the Airbnb configuration is to ensure that we write our code in line with a predefined standard. kindly remove all Eslint disabling comments and adjust the codes accordingly